src/layouts/BasicLayout.js

```diff
@@ -280,19 +280,19 @@ class BasicLayout extends React.PureComponent {
                 {
                   key: 'Pbench Dashboard',
                   title: 'Pbench Dashboard',
-                  href: 'https://example.com',
+                  href: 'http://pbench.perf.lab.eng.bos.redhat.com/dashboard/',
```

Comment:
Isn't this one is relative to the deployment?  If so we need to use the loaded configuration to construct the URL.  Since this should be a different URL for production, vs. staging, vs. local laptop deployment.